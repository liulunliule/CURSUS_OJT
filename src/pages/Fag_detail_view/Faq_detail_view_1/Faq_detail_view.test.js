import React from 'react';
import renderer from 'react-test-renderer';
import '@testing-library/jest-dom/extend-expect';
import { MemoryRouter } from 'react-router-dom'; // Import MemoryRouter
import Faq_detail_view from '.';

test('test Faq_detail_view', () => {
  const tree = renderer.create(
      <MemoryRouter>
        <Faq_detail_view/>
      </MemoryRouter>
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
