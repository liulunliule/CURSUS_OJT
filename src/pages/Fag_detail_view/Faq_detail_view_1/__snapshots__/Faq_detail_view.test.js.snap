// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test Faq_detail_view 1`] = `
<div
  className="payments"
>
  <div
    className="payments-container"
  >
    <div
      className="payments__inner"
    >
      <div
        className="payments__toolbar"
      >
        <div
          className="payments__toolbar-home"
        >
          <a
            href="/"
            onClick={[Function]}
          >
            Home 
          </a>
          / 
          <a
            href="/"
            onClick={[Function]}
          >
            Help 
          </a>
          /Payments
        </div>
        <div
          className="payments__toolbar-search"
        >
          <div
            className="payments__toolbar-icon"
          >
            <span
              aria-label="search"
              className="anticon anticon-search"
              role="img"
            >
              <svg
                aria-hidden="true"
                data-icon="search"
                fill="currentColor"
                focusable="false"
                height="1em"
                viewBox="64 64 896 896"
                width="1em"
              >
                <path
                  d="M909.6 854.5L649.9 594.8C690.2 542.7 712 479 712 412c0-80.2-31.3-155.4-87.9-212.1-56.6-56.7-132-87.9-212.1-87.9s-155.5 31.3-212.1 87.9C143.2 256.5 112 331.8 112 412c0 80.1 31.3 155.5 87.9 212.1C256.5 680.8 331.8 712 412 712c67 0 130.6-21.8 182.7-62l259.7 259.6a8.2 8.2 0 0011.6 0l43.6-43.5a8.2 8.2 0 000-11.6zM570.4 570.4C528 612.7 471.8 636 412 636s-116-23.3-158.4-65.6C211.3 528 188 471.8 188 412s23.3-116.1 65.6-158.4C296 211.3 352.2 188 412 188s116.1 23.2 158.4 65.6S636 352.2 636 412s-23.3 116.1-65.6 158.4z"
                />
              </svg>
            </span>
          </div>
          <input
            className="payments__toolbar-input"
            id=""
            name=""
            placeholder="Search"
            type="text"
          />
        </div>
      </div>
      <h1
        className="payments__title"
      >
        Payments
      </h1>
    </div>
  </div>
  <div
    className="payments__main"
  >
    <div
      className="payments-container"
    >
      <div
        className="payments__catalogue"
      >
        <h4
          className="payments__catalogue-title"
        >
          Instructor Topics
        </h4>
        <ul>
          <li
            className="payments__catalogue-item"
          >
            <div
              className="payments__catalogue-item-icon"
            >
              <svg
                aria-hidden="true"
                className="svg-inline--fa fa-cart-shopping "
                data-icon="cart-shopping"
                data-prefix="fas"
                focusable="false"
                role="img"
                style={Object {}}
                viewBox="0 0 576 512"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M0 24C0 10.7 10.7 0 24 0L69.5 0c22 0 41.5 12.8 50.6 32l411 0c26.3 0 45.5 25 38.6 50.4l-41 152.3c-8.5 31.4-37 53.3-69.5 53.3l-288.5 0 5.4 28.5c2.2 11.3 12.1 19.5 23.6 19.5L488 336c13.3 0 24 10.7 24 24s-10.7 24-24 24l-288.3 0c-34.6 0-64.3-24.6-70.7-58.5L77.4 54.5c-.7-3.8-4-6.5-7.9-6.5L24 48C10.7 48 0 37.3 0 24zM128 464a48 48 0 1 1 96 0 48 48 0 1 1 -96 0zm336-48a48 48 0 1 1 0 96 48 48 0 1 1 0-96z"
                  fill="currentColor"
                  style={Object {}}
                />
              </svg>
            </div>
             
            <a
              className=""
              href="/secondLayout/Faq_detail_view-1"
              onClick={[Function]}
            >
              Payments
            </a>
          </li>
          <li
            className="payments__catalogue-item"
          >
            <div
              className="payments__catalogue-item-icon"
            >
              <svg
                aria-hidden="true"
                className="svg-inline--fa fa-sellsy "
                data-icon="sellsy"
                data-prefix="fab"
                focusable="false"
                role="img"
                style={Object {}}
                viewBox="0 0 640 512"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M539.71 237.308c3.064-12.257 4.29-24.821 4.29-37.384C544 107.382 468.618 32 376.076 32c-77.22 0-144.634 53.012-163.02 127.781-15.322-13.176-34.934-20.53-55.157-20.53-46.271 0-83.962 37.69-83.962 83.961 0 7.354.92 15.015 3.065 22.369-42.9 20.225-70.785 63.738-70.785 111.234C6.216 424.843 61.68 480 129.401 480h381.198c67.72 0 123.184-55.157 123.184-123.184.001-56.384-38.916-106.025-94.073-119.508zM199.88 401.554c0 8.274-7.048 15.321-15.321 15.321H153.61c-8.274 0-15.321-7.048-15.321-15.321V290.626c0-8.273 7.048-15.321 15.321-15.321h30.949c8.274 0 15.321 7.048 15.321 15.321v110.928zm89.477 0c0 8.274-7.048 15.321-15.322 15.321h-30.949c-8.274 0-15.321-7.048-15.321-15.321V270.096c0-8.274 7.048-15.321 15.321-15.321h30.949c8.274 0 15.322 7.048 15.322 15.321v131.458zm89.477 0c0 8.274-7.047 15.321-15.321 15.321h-30.949c-8.274 0-15.322-7.048-15.322-15.321V238.84c0-8.274 7.048-15.321 15.322-15.321h30.949c8.274 0 15.321 7.048 15.321 15.321v162.714zm87.027 0c0 8.274-7.048 15.321-15.322 15.321h-28.497c-8.274 0-15.321-7.048-15.321-15.321V176.941c0-8.579 7.047-15.628 15.321-15.628h28.497c8.274 0 15.322 7.048 15.322 15.628v224.613z"
                  fill="currentColor"
                  style={Object {}}
                />
              </svg>
            </div>
             
            <a
              className=""
              href="/secondLayout/Faq_detail_view-2"
              onClick={[Function]}
            >
              Selling & Promotion
            </a>
          </li>
          <li
            className="payments__catalogue-item"
          >
            <div
              className="payments__catalogue-item-icon"
            >
              <svg
                aria-hidden="true"
                className="svg-inline--fa fa-cart-shopping "
                data-icon="cart-shopping"
                data-prefix="fas"
                focusable="false"
                role="img"
                style={Object {}}
                viewBox="0 0 576 512"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M0 24C0 10.7 10.7 0 24 0L69.5 0c22 0 41.5 12.8 50.6 32l411 0c26.3 0 45.5 25 38.6 50.4l-41 152.3c-8.5 31.4-37 53.3-69.5 53.3l-288.5 0 5.4 28.5c2.2 11.3 12.1 19.5 23.6 19.5L488 336c13.3 0 24 10.7 24 24s-10.7 24-24 24l-288.3 0c-34.6 0-64.3-24.6-70.7-58.5L77.4 54.5c-.7-3.8-4-6.5-7.9-6.5L24 48C10.7 48 0 37.3 0 24zM128 464a48 48 0 1 1 96 0 48 48 0 1 1 -96 0zm336-48a48 48 0 1 1 0 96 48 48 0 1 1 0-96z"
                  fill="currentColor"
                  style={Object {}}
                />
              </svg>
            </div>
             
            <a
              href="/term"
              onClick={[Function]}
            >
              Quality Standards
            </a>
          </li>
          <li
            className="payments__catalogue-item"
          >
            <div
              className="payments__catalogue-item-icon"
            >
              <svg
                aria-hidden="true"
                className="svg-inline--fa fa-ruler "
                data-icon="ruler"
                data-prefix="fas"
                focusable="false"
                role="img"
                style={Object {}}
                viewBox="0 0 512 512"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M177.9 494.1c-18.7 18.7-49.1 18.7-67.9 0L17.9 401.9c-18.7-18.7-18.7-49.1 0-67.9l50.7-50.7 48 48c6.2 6.2 16.4 6.2 22.6 0s6.2-16.4 0-22.6l-48-48 41.4-41.4 48 48c6.2 6.2 16.4 6.2 22.6 0s6.2-16.4 0-22.6l-48-48 41.4-41.4 48 48c6.2 6.2 16.4 6.2 22.6 0s6.2-16.4 0-22.6l-48-48 41.4-41.4 48 48c6.2 6.2 16.4 6.2 22.6 0s6.2-16.4 0-22.6l-48-48 50.7-50.7c18.7-18.7 49.1-18.7 67.9 0l92.1 92.1c18.7 18.7 18.7 49.1 0 67.9L177.9 494.1z"
                  fill="currentColor"
                  style={Object {}}
                />
              </svg>
            </div>
             
            <a
              href="/term"
              onClick={[Function]}
            >
              Course Building
            </a>
          </li>
          <li
            className="payments__catalogue-item"
          >
            <div
              className="payments__catalogue-item-icon"
            >
              <svg
                aria-hidden="true"
                className="svg-inline--fa fa-bars-progress "
                data-icon="bars-progress"
                data-prefix="fas"
                focusable="false"
                role="img"
                style={Object {}}
                viewBox="0 0 512 512"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M448 160l-128 0 0-32 128 0 0 32zM48 64C21.5 64 0 85.5 0 112l0 64c0 26.5 21.5 48 48 48l416 0c26.5 0 48-21.5 48-48l0-64c0-26.5-21.5-48-48-48L48 64zM448 352l0 32-256 0 0-32 256 0zM48 288c-26.5 0-48 21.5-48 48l0 64c0 26.5 21.5 48 48 48l416 0c26.5 0 48-21.5 48-48l0-64c0-26.5-21.5-48-48-48L48 288z"
                  fill="currentColor"
                  style={Object {}}
                />
              </svg>
            </div>
             
            <a
              href="/term"
              onClick={[Function]}
            >
              Course Management
            </a>
          </li>
          <li
            className="payments__catalogue-item"
          >
            <div
              className="payments__catalogue-item-icon"
            >
              <svg
                aria-hidden="true"
                className="svg-inline--fa fa-helmet-safety "
                data-icon="helmet-safety"
                data-prefix="fas"
                focusable="false"
                role="img"
                style={Object {}}
                viewBox="0 0 576 512"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M256 32c-17.7 0-32 14.3-32 32l0 2.3 0 99.6c0 5.6-4.5 10.1-10.1 10.1c-3.6 0-7-1.9-8.8-5.1L157.1 87C83 123.5 32 199.8 32 288l0 64 512 0 0-66.4c-.9-87.2-51.7-162.4-125.1-198.6l-48 83.9c-1.8 3.2-5.2 5.1-8.8 5.1c-5.6 0-10.1-4.5-10.1-10.1l0-99.6 0-2.3c0-17.7-14.3-32-32-32l-64 0zM16.6 384C7.4 384 0 391.4 0 400.6c0 4.7 2 9.2 5.8 11.9C27.5 428.4 111.8 480 288 480s260.5-51.6 282.2-67.5c3.8-2.8 5.8-7.2 5.8-11.9c0-9.2-7.4-16.6-16.6-16.6L16.6 384z"
                  fill="currentColor"
                  style={Object {}}
                />
              </svg>
            </div>
             
            <a
              href="/term"
              onClick={[Function]}
            >
              Trust & Safety
            </a>
          </li>
        </ul>
      </div>
      <div
        className="payments__content"
      >
        <div
          className="payment__content-1"
        >
          <h4
            className="payments__content-title"
          >
            <b>
              Taxes
            </b>
          </h4>
          <ul
            className="payments__content-desc"
          >
            <li
              className="payments__content-desc-item"
            >
              <a
                href="/"
                onClick={[Function]}
              >
                Common Instructor Questions About The W-8BEN
              </a>
            </li>
            <li
              className="payments__content-desc-item"
            >
              <a
                href="/"
                onClick={[Function]}
              >
                Taxes: What do Instructors Need to Submit to Cursus?
              </a>
            </li>
            <li
              className="payments__content-desc-item"
            >
              <a
                href="/"
                onClick={[Function]}
              >
                What Does Cursus do With my Tax Information (i.e. Withholding)?
              </a>
            </li>
            <li
              className="payments__content-desc-item"
            >
              <a
                href="/"
                onClick={[Function]}
              >
                How do I Submit my Tax Forms to Cursus?
              </a>
            </li>
            <li
              className="payments__content-desc-item"
            >
              <a
                href="/"
                onClick={[Function]}
              >
                How Does Cursus Handle VAT / GST?
              </a>
            </li>
          </ul>
        </div>
        <div
          className="payment__content-2"
        >
          <h4
            className="payments__content-title"
          >
            <b>
              Payments - General
            </b>
          </h4>
          <ul
            className="payments__content-desc"
          >
            <li
              className="payments__content-desc-item"
            >
              <a
                className="payments__content-desc-link"
                href="/"
                onClick={[Function]}
              >
                How do I Earn Revenue From Cursus for Business?
              </a>
            </li>
            <li
              className="payments__content-desc-item"
            >
              <a
                href="/"
                onClick={[Function]}
              >
                How to opt out of Cursus for Business
              </a>
            </li>
            <li
              className="payments__content-desc-item"
            >
              <a
                href="/"
                onClick={[Function]}
              >
                Payoneer FAQ
              </a>
            </li>
            <li
              className="payments__content-desc-item"
            >
              <a
                href="/"
                onClick={[Function]}
              >
                Missing Instructor Payments
              </a>
            </li>
            <li
              className="payments__content-desc-item"
            >
              <a
                href="/"
                onClick={[Function]}
              >
                Instructor Revenue Share
              </a>
            </li>
            <li
              className="payments__content-desc-item"
            >
              <a
                href="/"
                onClick={[Function]}
              >
                Instructor Revenue Report
              </a>
            </li>
            <li
              className="payments__content-desc-item"
            >
              <a
                href="/"
                onClick={[Function]}
              >
                Instructor Payment Overview
              </a>
            </li>
          </ul>
        </div>
      </div>
    </div>
  </div>
</div>
`;
